website/docs/advanced/audit-log.md

```diff
@@ -3,16 +3,16 @@ id: audit_log
 title: The audit log
 ---
 
-When something is not working as expected it is important to be able to track what changed when, and who performed the change.
+The audit log lets you track changes in Unleash. It lists _what_ changed _when_ it changed, and _who_ performed the change.
```

Comment:
```suggestion
The audit log lets you track changes in Unleash. It lists _what_ changed, _when_ it changed, and _who_ performed the change.
```

Reply:
Also, kudos on the correct use of Oxford comma :)

Reply:
Thanks! 😄 And thanks for noticing!